yaml/screenconnect.yaml

```diff
@@ -56,7 +56,17 @@ Artifacts:
   - File: C:\ProgramData\ScreenConnect Client*\user.config
     Description: ScreenConnect client user configuration
     OS: Windows
-  EventLog: []
+  EventLog:
+  - EventID: 7045
+    ProviderName: ["ScreenConnect", "ScreenConnect Client (<hex string>)"]
+    LogFile: Application.evtx
+    ServiceName: ScreenConnect Client (<hex string>)
+    Description: Service installation event as a result of ScreenConnect installation.
+  - EventID: 20
+    ProviderName: ["ScreenConnect", "ScreenConnect Client (<hex string>)"]
+    LogFile: Application.evtx
+    ServiceName: ScreenConnect Client (<hex string>)
```

Comment:
I think the YAML verification is failing because of the "(hex string)" part, try to remove it?

Reply:
Removed <hex string> and standardized provider as "ScreenConnect Client ()"